### PR TITLE
Upcoming and planned languages pages improvement

### LINKED ISSUE #3149: Incorrect sections on upcoming and planned language pages
Languages such as [PowerShell](http://exercism.io/languages/powershell) that don't yet have enough exercises to launch are listed under the **Upcoming** header on [/languages](http://exercism.io/languages). 

There are two issues:

* On the main page it displays the same text twice.
* On the other pages linked to from the sidebar it displays that same text (but only once) instead of linking to the content for that section.

The same happens for a planned language, e.g. [Pascal](http://exercism.io/languages/pascal)

The routes file for these pages is [app/routes/languages.rb](https://github.com/exercism/exercism.io/blob/master/app/routes/languages.rb).

- https://github.com/exercism/exercism.io/blob/master/app/routes/languages.rb#L24-L38
- https://github.com/exercism/exercism.io/blob/master/app/routes/languages.rb#L49-L64

These then call the template with ` erb :"languages/language"`: https://github.com/exercism/exercism.io/blob/master/app/views/languages/language.erb

### PR DESCRIPTION
**Pull request by MohanL**
Fixes #3149

This PR fixes the bug referenced in the issue #3149.
Currently the bug is caused by the logic which controls when to render the launch page. 
This PR fixes the logic error by adding conditions to determine when to render the launch page.

### CODE REVIEW

File: app/views/languages/language.erb [hunk 1]
```diff
@@ -71,11 +71,16 @@
...
-        </div>
+        <% if topic == "launch" %>
+          <div class="active tab-pane" id="launch">
+            <%= md docs.launch %>
+          </div>
+        <% else %>
+          <%= erb :"languages/_#{topic}", locals: { track: track, docs: docs } %>
+        <% end %>
+      <% else %>
+        <%= erb :"languages/_#{topic}", locals: { track: track, docs: docs } %>
```
Comment (bernardoamc) [inline app/views/languages/language.erb]: Is this the same as line `79`? If so we could do something like:

```ruby
<% if !track.active &&  topic == "launch" %>
  <div class="active tab-pane" id="launch">
    <%= md docs.launch %>
  </div>
<% else %>
  <%= erb :"languages/_#{topic}", locals: { track: track, docs: docs } %>
<% end %>
```

**Comment by bernardoamc** [comment]
Awesome @MohanL ! ❤️

Could you post screenshots showing how is the page before and after the fix?

File: app/views/languages/language.erb (hunk 1 above)
Comment (MohanL) [inline app/views/languages/language.erb]: Sure，that is definitely a good idea. Thank you ! @bernardoamc

**Comment by MohanL** [comment]
Content duplicate 
Before
image

After
image

**Comment by MohanL** [comment]
CSS style 
Before
image

After 
image

**Comment by bernardoamc** [comment]
Wondering if we should leave that `Installing` section empty or if we should keep the `Help Us Launch` section. What's your opinion @MohanL ?

**Comment by MohanL** [comment]
@bernardoamc 
I feel like no duplicate content on the installing section would be better. 
Can we hide these sections for the non-active languages ?

**Comment by bernardoamc** [comment]
> Can we hide these sections for a non-active language ? and render them when the language is active ?

This seems like a good idea, but I would like to validate this with UX/Front End developers. :)

@kytrinyx Who could help us with that? Do you have a strong opinion about this topic?

**Comment by kytrinyx** [comment]
> Can we hide these sections for a non-active language 

If the documentation exists, we should show it, I think. Also, if the documentation doesn't exist, the old version used to show you an encouragement to go contribute that documentation. I like the idea of that.

This is already so much better, that I think I'll just add a couple of small tweaks on top of what you've done and pull it in. Then we can think about iterating on it afterwards (but still get this shipped into production).

**Comment by kytrinyx** [comment]
Thank you so much! :heart:

**Pull request merged**